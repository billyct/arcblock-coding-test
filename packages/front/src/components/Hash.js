import cn from 'classnames'

const Hash = ({children, className}) => (
  <span
    className={cn(`text-blue-500 cursor-pointer w-80 truncate text-sm hover:underline`, className)}
  >
    {children}
  </span>
)


export default Hash
