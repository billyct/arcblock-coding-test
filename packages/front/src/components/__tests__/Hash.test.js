import {render} from '@testing-library/react'

import Hash from '../Hash'

test('Hash Component', () => {
  const {container} = render(<Hash>29d6a3540acfa0a950bef2bfdc75cd51c24390fd</Hash>)
  expect(container).toMatchSnapshot()
})
