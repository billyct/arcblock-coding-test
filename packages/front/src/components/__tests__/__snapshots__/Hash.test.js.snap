// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hash Component 1`] = `
<div>
  <span
    class="text-blue-500 cursor-pointer w-80 truncate text-sm hover:underline"
  >
    29d6a3540acfa0a950bef2bfdc75cd51c24390fd
  </span>
</div>
`;
